lib/components/MegaMenuLayout/MegaMenuLayout.tsx

```diff
@@ -43,7 +43,7 @@ const MegaMenuLayout: React.FC<MegaMenuWrapperProps> = ({
     <>
       <div className="relative z-10 flex w-full items-center justify-center sm:h-[120px]">
         <nav
-          className="flex h-full w-full items-center justify-between gap-x-4 overflow-hidden px-0"
+          className="flex h-full w-full items-center justify-between gap-x-4 px-0"
```

Comment:
CC: @Harry-Ross 

Will this effect anything on the Website?

Reply:
🤦‍♀️ Yes. After doing some more manual tests I found it doesn't work well on mobile. I should make the changes on Rules instead.. Thanks for pointing this out!